library/mimIcon/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+
+include_directories(${QT_INCLUDES} src)
```

Comment:
You don't need the QT_INCLUDES here or anywhere else as you link against the correct Qt Targets already